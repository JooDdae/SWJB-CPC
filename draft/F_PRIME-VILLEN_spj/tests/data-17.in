2633501110338926472113316927292377433926639391757771096359617710979331249629917873372131349359377936553823755936238807437313034018101947322738747562398592797919958515303578394193593459754772375378677211660790912161250993709437371776829972134723336133191316397672357106157012411979281514774172969883412991994007884963795477785318675217514327826978299767772714939619715146518771237
