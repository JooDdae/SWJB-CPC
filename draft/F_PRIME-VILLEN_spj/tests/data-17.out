202
56 2 672 27 2 24 1 118 1 0 0 2 449 1 4 10 1 2 4 4 593 386 1161 4 3 4 4 4 29 828 18 4 4 29 22 2 2 204 819 0 138 2 2 4 1 0 2 0 70 2 17 2 4 22 2 847 143 4 3 17 507 1097 14 4 2 213 79 1019 161 2 1 1 4 2 1091 739 1217 1 22 22 46 769 703 3 4 146 81 2 17 2 622 3 15 4 1 2 4 3 2 993 4 1 0 0 854 1128 1 0 18 1 681 2 127 14 4 2 4 0 4 4 879 25 1 0 2 15 1 2 2 2 18 2 2 8 0 2 38 25 19 1 2 3 4 178 3 126 1 13 8 22 60 3 0 15 4 13 4 428 153 13 10 1140 553 1103 832 3 15 4 4 147 285 3 1 0 4 3 0 14 1 4 1037 4 145 25 19 4 4 1 4 35 2 1187 4 0 3 0 629 151 0 1 2 4 